spec/services/notifications/welcome_notification/send_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe Notifications::WelcomeNotification::Send, type: :service do
+  subject(:welcome_notification_service) { described_class }
+
+  let(:user) { create(:user) }
+  let(:dev_account) { create(:user) }
+  let(:welcome_broadcast) { create(:broadcast, :onboarding) }
+  let(:welcome_notification) { welcome_notification_service.call(user.id, welcome_broadcast) }
+
+  describe "::call" do
+    before do
+      allow(User).to receive(:dev_account).and_return(dev_account)
+    end
+
+    it "checks a newly created welcome notification", :aggregate_failures do
```

Comment:
Not a strict one: here is only one test, so `let`s are not that useful. I think the test would be easier to understand if the service was called in the `it` block.

Reply:
Yes, I did not notice that one :). I have removed the lets.